### DISCUSSION TITLE
Build log Bug

### DISCUSSION THREAD

**Discussion by Murcila** [alyssapiresfernandescefet/openrpg#18] [Q&A]
Eu tive um problema, eu realizei tudo o que pediu (excepto utilizar o Random_org_key). E quando fui fazer o "Deploy Branch", ocorreu um erro na build log, algo haver com Node.js eu acho. Tentei realizar outro app fazendo os mesmos procedimentos e ainda acontece este erro. Tem algum jeito de consertar isto? Se este jeito for muito complicado, peço desculpas.
![image](https://user-images.githubusercontent.com/103331940/187045108-eaa58ad2-ac51-404f-a3ab-a3215314a31b.png)

**Comment by lucaspiresfernandes** [comment]
Olá, @Murcila! Esse tipo de problema é causado por algum passo errado na criação do App. Pra esse caso, precisamos resolver esse problema em um contato mais direto. Me chama no Discord: That Alyssa#0615
